src/data/screenshots.json

```diff
@@ -268,7 +268,7 @@
                 "anchor": "ios_registertest",
                 "images": [
                     { "filename": "/assets/screenshots/2-6/en/ios/iPhone 11 Pro-tan_submissionflow_qr_0002", "alt": "Test registration & Key sharing image 1 of 11" },
-                    { "filename": "/assets/screenshots/2-6/en/ios/iPhone 11 Pro-tan_submissionflow_tan_0003", "alt": "Test registration & Key sharing image 2 of 11" },
+                    { "filename": "/assets/screenshots/2-5/en/ios/iPhone 11 Pro-tan_submissionflow_tan_0003", "alt": "Test registration & Key sharing image 2 of 11" },
```

Comment:
What is the reason for this change? I don't see any obvious difference between the screenshots:
|2.5|2.6|2.7|
|-|-|-|
|![2.5](https://raw.githubusercontent.com/corona-warn-app/cwa-website/master/src/assets/screenshots/2-5/en/ios/iPhone%2011%20Pro-tan_submissionflow_tan_0003.png)|![2.6](https://raw.githubusercontent.com/corona-warn-app/cwa-website/master/src/assets/screenshots/2-6/en/ios/iPhone%2011%20Pro-tan_submissionflow_tan_0003.png)|![2.7](https://raw.githubusercontent.com/corona-warn-app/cwa-website/master/src/assets/screenshots/2-7/en/ios/iPhone%2011%20Pro-tan_submissionflow_tan_0003.png)|

Though for the German screenshots (referenced below) the keyboard is indeed broken:
|2.5|2.6|2.7|
|-|-|-|
|![2.5](https://raw.githubusercontent.com/corona-warn-app/cwa-website/master/src/assets/screenshots/2-5/de/ios/iPhone%2011%20Pro-tan_submissionflow_tan_0003.png)|![2.6](https://raw.githubusercontent.com/corona-warn-app/cwa-website/master/src/assets/screenshots/2-6/de/ios/iPhone%2011%20Pro-tan_submissionflow_tan_0003.png)|![2.7](https://raw.githubusercontent.com/corona-warn-app/cwa-website/master/src/assets/screenshots/2-7/de/ios/iPhone%2011%20Pro-tan_submissionflow_tan_0003.png)|

However, wouldn't it make sense then to delete the unused screenshots to avoid confusion?